packages/playground/compile-wordpress/Dockerfile

```diff
@@ -37,40 +63,57 @@ RUN git clone https://github.com/WordPress/sqlite-database-integration.git \
     # Required by the SQLite integration plugin:
     cp wordpress/wp-config-sample.php wordpress/wp-config.php
 
+# Enable the SQLite support for the src folder in wordpress-develop:
+
+RUN if [[ -d wordpress-develop-trunk ]] ; then \
```

Comment:
This RUN is mostly the same as the one above, perhaps merging them and exporting a PATH_PREFIX inside of an `if/else` would make it shorter? I'm happy to fiddle with that if you invite me to your fork.

Reply:
You should already have access according to the "Allow edits by maintainers" checkbox, so feel free to make any edits